src/Application.php

```diff
@@ -227,6 +243,10 @@ public function run(): int
                 $message = str_replace($basePath, '', $message);
             }
 
+            $container
```

Comment:
seen as we're catching any and all throwables it seemed like a logical place to fire of some cleanup